packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -70,6 +70,7 @@ export function getColumns(data: DataFrame, availableWidth: number, columnMinWid
       sortType: selectSortType(field.type),
       width: fieldTableOptions.width,
       minWidth: 50,
+      filter: filterByValue,
```

Comment:
Hm, from react-table docs: [If a function is passed, it must be memoized](https://react-table.tanstack.com/docs/api/useFilters#column-options). Haven't noticed any weird behaviour but I guess this is a purely perf optimization they suggest?

Reply:
Yes, I read that but I think it is as `getColumns` is memoized? WDYT?

Reply:
I memoized it :)